### PR TITLE
Tick version to 0.2

### PR DESCRIPTION
**Pull request by pmlandwehr**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
 
 build:
-  skip: True  # [win and py27]
-  number: {{ build_num }}
+  number: {{ build }}
   script: python setup.py install --single-version-externally-managed --record=record.txt
   features:
-    - vc14  # [win and py35]
+    - vc 9  # [win and py27]
+    - vc10  # [win and py35]
```
Comment (rmax) [inline recipe/meta.yaml:22]: Is this correct? I understood vc14 is used for py35 and py36 and vc10 was used for py34.

**Comment by rmax** [comment]
It seems CI builds didn't get trigger :-/

**Comment by pmlandwehr** [comment]
Yeah, I was unclear on why they didn't get started. Not sure of an easy fix...

File: recipe/meta.yaml (hunk 1 above)
Comment (pmlandwehr) [reply recipe/meta.yaml:22]: I'll buy that; my change was based on [conda-forge's docs](https://conda-forge.org/docs/buildwin.html#vc-features). Do you have a link? Be nice to put in the recipe...

Comment (rmax) [reply recipe/meta.yaml:22]: From the docs:
```
build:
  features:
    - vc9   # [win and py27]
    - vc10  # [win and py34]
    - vc14  # [win and py>=35]
```
We can disregard the `py34` line as conda does not build for that version.

**Comment by rmax** [comment]
Could you try to rebase and force push to trigger the build again?

**Comment by rpanai** [comment]
When it's going to be available on conda-forge?

**Comment by rmax** [comment]
Fixed here https://github.com/conda-forge/fbprophet-feedstock/pull/6 

Let's wait the master builds.

**Pull request merged**